settings/src/main/scala/org/typelevel/sbt/TypelevelSettingsPlugin.scala

```diff
@@ -194,6 +194,14 @@ object TypelevelSettingsPlugin extends AutoPlugin {
           }
         }
     },
+    Compile / doc / scalacOptions ++= {
+      scalaVersion.value match {
+        case V(V(3, minor, Some(build), _)) if minor >= 2 && build >= 1 =>
```

Comment:
wait a sec, it wouldn't be applied in case of `3.x.0` even with x > 3, right? For example 3.3.0 falls out of this condition.

Reply:
I would add something like
```scala
case V(V(3, 2, Some(build), _)) if build >= 1 => Seq("-Ygenerate-inkuire")
case V(V(3, minor, ), _)) if minor >= 2 => Seq("-Ygenerate-inkuire")
```
or some alternative syntax to express both cases

Reply:
Ah yeah, true! Thank you for spotting that

Reply:
Woops, thanks! :)

Reply:
🙇